test/utils/image/manifest.go

```diff
@@ -219,8 +219,8 @@ func initImageConfigs() map[int]Config {
 	configs[InvalidRegistryImage] = Config{invalidRegistry, "alpine", "3.1"}
 	configs[IpcUtils] = Config{e2eRegistry, "ipc-utils", "1.0"}
 	configs[JessieDnsutils] = Config{e2eRegistry, "jessie-dnsutils", "1.0"}
-	configs[Kitten] = Config{e2eRegistry, "kitten", "1.0"}
-	configs[Nautilus] = Config{e2eRegistry, "nautilus", "1.0"}
+	configs[Kitten] = Config{e2eRegistry, "kitten", "1.4"}
+	configs[Nautilus] = Config{e2eRegistry, "nautilus", "1.4"}
```

Comment:
Unfortunately, kitten and nautilus 1.4 haven't been pushed to either e2eRegistry nor promoterE2eRegistry